workshops/management/commands/fake_database.py

```diff
@@ -39,10 +39,18 @@ def fake_airports(self, faker, count=5):
 
     def fake_roles(self, faker):
         """Provide fixed roles (before they end up in fixtures, see #626)."""
-        roles = ['helper', 'instructor', 'host', 'learner', 'organizer',
-                 'tutor', 'debriefed']
-        for role in roles:
-            Role.objects.create(name=role)
+        roles = [
+            ('debriefed', 'Debriefed', False),
```

Comment:
Why is "Debriefed" a role?  Also, I don't understand the doc string on this method.